faasmcli/faasmcli/tasks/format_code.py

```diff
@@ -51,7 +51,12 @@ def format(ctx, check=False):
         .split("\n")[:-1]
     )
 
-    clang_cmd = "clang-format-10 -i {}".format(" ".join(files_to_check))
+    if check:
```

Comment:
In the process of doing this PR, I realised that the formatting task was not behaving as expected with the `--check` flag. With this changes running `inv format-code --check` will have `clang-format-10` return an error if formatting changes are needed.